src/gps/GPS.cpp

```diff
@@ -576,6 +567,8 @@ int32_t GPS::runOnce()
             return 0;
         }
     }
+    if (config.position.gps_enabled == false)
+        return 0;
```

Comment:
Can we `return disable();` here, or do we leave the thread open for toggling?

Reply:
Correct, this is so the triple-click will work. I'm not opposed to adding a re-enable function to go with the disable(), but that seems out of scope for this patch set.